src/components/Record.js

```diff
@@ -3,13 +3,22 @@ import moment from 'moment';
 export default function Record({ className, title, createdAt, type, text }) {
     const formattedTime = moment(createdAt).calendar();
 
+    let renderedText = text;
+    if (text.startsWith('http')) {
+        renderedText = (
+            <a className="link" href="text">
+                {text}
+            </a>
+        );
+    }
```

Comment:
Pulling the record text into a link if it's a url - just makes it a nicer experience to interact with